src/index.scss

```diff
@@ -1,10 +1,24 @@
-$feature-flags: (
-  ui-shell: true,
-  grid-columns-16: true,
-);
+// $feature-flags: (
+//   ui-shell: true,
+//   grid-columns-16: true,
+// );
 
```

Comment:
You need these classes - so that your layout matches the expected outcome.

Reply:
Kelly, 
Thank you! Not sure why I commented that part out! 
Fixed it and redeployed it to the cloud.
- https://carbon-tutorial-alevi-ibm.mybluemix.net/
- https://deploy-preview-1495--carbon-tutorial.netlify.com/#/